dist/src/test/java/io/camunda/application/CamundaDockerIT.java

```diff
@@ -103,6 +108,75 @@ public void testStartCamundaDocker() throws Exception {
     }
   }
 
+  @Test
+  public void testStartStandaloneCamundaWithUnifiedConfiguration()
```

Comment:
:thinking: I wonder if we should move this test to QA, because that is where this fits better.